cluster/gce/config-default.sh

```diff
@@ -177,6 +174,12 @@ HEAPSTER_MACHINE_TYPE="${HEAPSTER_MACHINE_TYPE:-}"
 # NON_MASTER_NODE_LABELS are labels will only be applied on non-master nodes.
 NON_MASTER_NODE_LABELS="${KUBE_NON_MASTER_NODE_LABELS:-}"
 
+if [[ "${PREEMPTIBLE_MASTER}" == "true" ]]; then
+    NODE_LABELS="${NODE_LABELS},cloud.google.com/gke-preemptible=true"
+elif [[ "${PREEMPTIBLE_NODE}" == "true" ]]; then
```

Comment:
The logic here seems to imply that preemptible master means that nodes are also preemptible. Is that correct? Otherwise, we need to handle the case where the master is preemptible but nodes are not (Though I don't know why would someone have such a setup).

Reply:
There is no master only node label variable and I'd like to avoid defining a new variable for such a corner case.

Reply:
nit: perhaps add a comment here that we don't support PREEMPTIBLE_MASTER=true && PREEMPTIBLE_NODE=false settings here?